tree/dataframe/test/dataframe_snapshot.cxx

```diff
@@ -61,6 +61,102 @@ class RDFSnapshotMT : public ::testing::Test {
 };
 #endif // R__USE_IMT
 
+
+// Test for custom basket size in Snapshot
+class SnapshotCustomBasketRAII {
+private:
+   std::string fInputFile;
+   std::string fOutputFileCustom;
+   std::string fOutputFileCollection;
+   TFile fFile;
+   TTree fTree;
```

Comment:
Please move these data members inside the constructor body directly, no need to keep them around for the entire lifetime of the class instance

Reply:
In this case it's mostly to ensure smoother lifetime management. There is really no need to keep the file open for the entire lifetime of the class instance, it would also possibly lead to erroneous interactions with the `gSystem->Unlink` calls in the destructor.